model/src/main/kotlin/Project.kt

```diff
@@ -111,6 +111,13 @@ data class Project(
         )
     }
 
+    init {
+        require(scopeDependencies == null || dependencyGraph == null) {
+            "Not both 'scopeDependencies' and 'dependencyGraph' may be set, as otherwise it is ambiguous which one " +
```

Comment:
Maybe add a comma after both?

Reply:
Then you'd also need a comma after `'dependencyGraph'`. I think it's fine as-is.

Reply:
I have kept this as is to keep aligned with a similar message in `Provenance.kt` which I copied here.